### PR TITLE
Add firmware for NF10.

### PR DESCRIPTION
**Pull request by haydenroche5**
The firmware source code for NF9 and 10 is identical. NF10's main.cpp is just a symbolic link to NF9's main.cpp. Valve support is compiled in by defining the macro USE_VALVE to 1. It's compiled out by setting USE_VALVE to 0. The platform.ini build_flags set this macro accordingly for each project.

### CODE REVIEW

**Comment by paigen11** [comment]
@haydenroche5 would you mind resolving the conflicts in your branch and merging to the `nf10` branch for final review before it goes public?

File: valve-monitor/firmware/src/main.cpp [hunk 1]
```diff
@@ -29,14 +29,9 @@
...
 
-// Define USE_VALVE to compile in valve control support. If not defined, it's
-// turned on by default.
-#ifndef USE_VALVE
-#define USE_VALVE 1
-#endif
-
 // Replace with your product UID.
 // #define PRODUCT_UID "com.my-company.my-name:my-project"
+#define PRODUCT_UID "com.blues.hroche:quickstart"
```
Comment (m-mcgowan) [inline valve-monitor/firmware/src/main.cpp:34]: Can we leave this undefined by default, like this: https://github.com/blues/app-accelerators/blob/main/08-power-monitoring-and-control/firmware/src/app.h#L12

Comment (m-mcgowan) [reply valve-monitor/firmware/src/main.cpp:34]: Also please note that we have a short link, just in case the docs layout changes.

**Review by m-mcgowan — CHANGES_REQUESTED** [review]
LGTM - just the ProductUID thing.

**Review by m-mcgowan — APPROVED** [review]

**Pull request merged**